website_sale_product_category_seo/views/templates.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <template id="products_categories_cms" inherit_id="website_sale.products" name="Public Category CMS Block">
+        <xpath expr="//div[@id='products_grid']//*[1]" position="before">
+            <div t-if="category">
```

Comment:
i need the field to edit it in the frontend, if it is empty then i can not fill it